### PR TITLE
MKZ-575 Markurz Package Edition

### PR DESCRIPTION
**Pull request by FernandVEYRIER**
Markurz extension is now packed as a package allowing it to be reused on other projects.

### CODE REVIEW

File: src/rollup.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export * from "./components";
+export { default as markurzExtensionTheme } from "./theme";
```
Comment (alexandrevanhoutte) [inline src/rollup.ts:2]: I guess this is normal, but absolute import doesn't work isn't it ?

File: tsconfig.json [hunk 2]
```diff
@@ -15,11 +15,18 @@
...
     "moduleResolution": "node",
-    "resolveJsonModule": true,
-    "isolatedModules": true,
-    "noEmit": true,
-    "jsx": "react-jsx",
-    "baseUrl": "."
+//    "resolveJsonModule": true,
+//    "isolatedModules": true,
+//    "noEmit": true,
+//    "jsx": "react-jsx",
```
Comment (alexandrevanhoutte) [inline tsconfig.json:21]: maybe should be removed ?

File: rollup.config.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  //   ],
+  //   output: {
+  //     file: `dist/${packageJson.name}.min.js`,
+  //     format: "umd",
+  //     name: "markurz",
+  //     esModule: false,
+  //     exports: "named",
+  //     sourcemap: true,
+  //   },
+  // },
```
Comment (alexandrevanhoutte) [inline rollup.config.js:52]: Shouldn't not be removed ?

File: rollup.config.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import commonjs from "@rollup/plugin-commonjs";
+import typescript from "@rollup/plugin-typescript";
+import { dts } from "rollup-plugin-dts";
+// import { terser } from "rollup-plugin-terser";
+// import babel from "@rollup/plugin-babel";
+// import alias from "@rollup/plugin-alias";
+// import { fileURLToPath } from "url";
```
Comment (alexandrevanhoutte) [inline rollup.config.js:8]: Also comment

File: rollup.config.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+        exports: "named",
+      },
+    ],
+    external: ["react", "react-dom"],
+    plugins: [
+      // alias({
+      //   entries: {
+      //     src: fileURLToPath(new URL("src", import.meta.url)),
+      //   },
+      // }),
```
Comment (alexandrevanhoutte) [inline rollup.config.js:75]: also comment

**Review by alexandrevanhoutte — CHANGES_REQUESTED** [review]
mainly only comment

File: src/rollup.ts (hunk 1 above)
Comment (FernandVEYRIER) [reply src/rollup.ts:2]: Indeed, `index.ts` files all refer to each other according to relative paths, to allow the exports to be properly built.

**Review by alexandrevanhoutte — APPROVED** [review]

**Review by olzhik11 — APPROVED** [review]

**Pull request merged**